lib/server-impl.js

```diff
@@ -35,12 +40,24 @@ function createApp(options) {
         stores.clientMetricsStore
     );
 
+    if (config.importFromFile) {
+        await importDataFromFile(config);
+    }
+
     const server = app.listen({ port: options.port, host: options.host }, () =>
         logger.info(`Unleash started on port ${server.address().port}`)
     );
 
-    return new Promise((resolve, reject) => {
-        server.on('listening', () => resolve({ app, server, eventBus }));
+    return await new Promise((resolve, reject) => {
+        server.on('listening', () =>
+            resolve({
+                app,
+                server,
+                eventBus,
+                import: importData.bind(null, config),
+                export: exportData.bind(null, config),
```

Comment:
Why do you think it is important to expose import/export outside of Unleash? How do you think this should be used?

Reply:
as an alternative option to the admin API which requires authentication.

Reply:
in our specific use case, we have 
`features.yaml` which is an export from the production system
`features.overrides.yaml` git-tracked overrides
`features.local.yaml` non git-tracked overrides
the entrypoint merges all 3 and imports them into unleash.
maybe it should allow multiple `--importFile`'s ?

Reply:
on a second thought, I don't think allowing multiple `importFile`'s is a good idea. The logic for merging the export files is a bit displaced. This should be done by a user defined entrypoint.